app/pods/components/svg-component/component.js

```diff
@@ -0,0 +1,8 @@
+import Component from '@ember/component';
+
+/**
+ * SVG component for visualisation on the map.
+ */
+export default Component.extend({
+    tagName: '',
```

Comment:
fyi the styles.scss in this folder won't do anything as this component is tag less :)

Reply:
Will remove